wiki/Guides/Starting_a_Beatmap_Project/en.md

```diff
@@ -1,6 +1,6 @@
 # How to start a beatmap project
 
-*[Please specify project type (Discography, Album, Collab) by LuigiHann](https://osu.ppy.sh/community/forums/topics/12193)*
+*[Please specify project type (Discography, Album, Collab) by LuigiHann](https://web.archive.org/web/20120719062438/http://osu.ppy.sh/forum/t/12193)*
```

Comment:
it makes more sense to link directly to a pinned thread with rules: https://osu.ppy.sh/community/forums/topics/453937, with a caption along the lines of `*Make your project more descriptive by using a correct title*`